x/ecocredit/server/invariants.go

```diff
@@ -29,7 +29,7 @@ func tradableSupplyInvariant(store types.KVStore) (string, bool) {
 	)
```

Comment:
We actually need to update invariants, because now `tradable supply = SUM(user balances) + SUM(basket balances)`

Reply:
I'm not sure im following - if its just sending credits from one entity to another, doesn't the supply just stay the same? not new credits are being created?

Reply:
yes, but the invariant checks that all the balances = the supply. and now the invariant check will fail because we need to check basket balances too

Reply:
oh okay, i thought for a second you meant to change the TradableSupply in the store 😨 i'll update this function

Reply:
added basket check to invariants. feels kinda hacky though, let me know how it looks